wakebox/main.cpp

```diff
@@ -124,3 +188,90 @@ int main(int argc, char *argv[])
 
 	return res;
 }
+
+int main(int argc, char *argv[])
+{
+	unsigned int max_pairs = argc / 2;
+	char *tools[max_pairs];
+	char *binds[max_pairs];
```

Comment:
`std::vector<char*> tools(max_pairs, nullptr);`
`tools.data()` -> get the underlying C type

Reply:
(to avoid stack overflow on very large command lines)